<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGoodsIncomeDocDialog</class>
 <widget class="QDialog" name="EditGoodsIncomeDocDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="layoutProviderName">
       <item>
        <widget class="QLabel" name="lblProviderName">
         <property name="text">
          <string>Поставщик</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtProviderName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblCreateDate">
         <property name="text">
          <string>Дата создания</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dtCreateDate">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="tbNewGoods">
           <property name="toolTip">
            <string>Добавить товар</string>
           </property>
           <property name="toolTipDuration">
            <number>500</number>
           </property>
           <property name="text">
            <string>Добавить товар</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Icons/Icons/plus_add_green.ico</normaloff>:/Icons/Icons/plus_add_green.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tvGoods"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tbSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Icons/Icons/save_accept.ico</normaloff>:/Icons/Icons/save_accept.ico</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Icons/Icons/save_delete.ico</normaloff>:/Icons/Icons/save_delete.ico</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
